docker/entrypoint.sh

```diff
@@ -23,6 +23,10 @@ case "$1" in
             exec cargo "test" "--locked" "--release" "--features" "enable-sentry" "--workspace" "--" "--include-ignored"
         elif [[ "$TEST_FULL_SYNC" -eq "1" ]]; then
             exec cargo "test" "--locked" "--release" "--features" "enable-sentry" "--test" "acceptance" "--" "--nocapture" "--ignored" "full_sync_mainnet"
+        elif [[ "$TEST_DISK_REBUILD" -eq "1" ]]; then
+            exec cargo "test" "--locked" "--release" "--features" "enable-sentry,,test_sync_to_mandatory_checkpoint_${NETWORK }" "--manifest-path" "zebrad/Cargo.toml" "sync_to_mandatory_checkpoint_${NETWORK }"
+        elif [[ "$TEST_CHECKPOINT_SYNC" -eq "1" ]]; then
+            exec cargo "test" "--locked" "--release" "--features" "enable-sentry,,test_sync_past_mandatory_checkpoint_${NETWORK }" "--manifest-path" "zebrad/Cargo.toml" "test_sync_past_mandatory_checkpoint_${NETWORK }"
```

Comment:
Note: caching will be more efficient when we change from Rust features to environmental variables for these tests.

Until then, we just re-compile the test and `zebrad` binaries, which isn't too bad.